examples/src/index.js

```diff
@@ -57,17 +43,33 @@ function HitTestExample() {
   return <Box ref={ref} args={[0.1, 0.1, 0.1]} />
 }
 
-function App() {
+function ARExample() {
   return (
-    <VRCanvas>
-      <ambientLight intensity={0.5} />
-      <pointLight position={[5, 5, 5]} />
+    <Canvas frameloop="never" onCreated={({ gl }) => startXRloop(gl)}>
+      <XR buttonAR={true} sessionInit={{ requiredFeatures: ['hit-test'] }}>
```

Comment:
I think it'd be better if AR/VR buttons were a component instead of a prop, what do you think?